util/resource/resource_test.go

```diff
@@ -41,3 +62,21 @@ func TestSortForCreate(t *testing.T) {
 	g.Expect(sorted[0].GetKind()).To(BeIdenticalTo("Namespace"))
 	g.Expect(sorted[1].GetKind()).To(BeIdenticalTo("ConfigMap"))
 }
+
+func BenchmarkSort(b *testing.B) {
```

Comment:
Let's remove the benchmarking tests as it doesn't actually `test` for anything without being run manually.